trezorlib/client.py

```diff
@@ -1100,6 +1101,172 @@ def self_test(self):
+                    inflation_destination_account=op["inflation_destination"],
+                    clear_flags=op["clear_flags"],
+                    set_flags=op["set_flags"],
+                    master_weight=op["master_weight"],
+                    low_threshold=op["low_threshold"],
+                    medium_threshold=op["medium_threshold"],
+                    high_threshold=op["high_threshold"],
+                    home_domain=op["home_domain"],
+                    signer_type=op["signer_type"],
+                    signer_key=op["signer_key"],
+                    signer_weight=op["signer_weight"],
+                ))
+            # Change Trust
+            if op["type"] == 6:
+                asset = types.StellarAssetType(type=op["asset"]["type"], code=op["asset"]["code"],
+                                               issuer=op["asset"]["issuer"])
+                resp = self.call(proto.StellarChangeTrustOp(
+                    source_account=op["source_account"],
+                    limit=op["limit"],
+                    asset=asset
+                ))
+            # Allow Trust
+            if op["type"] == 7:
+                resp = self.call(proto.StellarAllowTrustOp(
+                    source_account=op["source_account"],
+                    trusted_account=op["trusted_account"],
+                    asset_type=op["asset_type"],
+                    asset_code=op["asset_code"],
+                    is_authorized=op["is_authorized"]
+                ))
+            # Merge Account
+            if op["type"] == 8:
+                resp = self.call(proto.StellarAccountMergeOp(
+                    source_account=op["source_account"],
+                    destination_account=op["destination_account"]
+                ))
+            # Manage data
+            if op["type"] == 10:
+                resp = self.call(proto.StellarManageDataOp(
+                    source_account=op["source_account"],
+                    key=op["key"],
+                    value=op["value"]
+                ))
+            # Merge Account
+            if op["type"] == 11:
+                resp = self.call(proto.StellarBumpSequenceOp(
+                    source_account=op["source_account"],
+                    bump_to=op["bump_to"]
+                ))
+
+            # Exit if the response was a StellarSignedTx
+            if resp.__class__.__name__ == "StellarSignedTx":
+                return resp
+
+        raise CallException("Reached end of operations without a signature")
+
+    @expect(proto.StellarMessageSignature)
+    def stellar_sign_message(self, address_n, message):
+        return self.call(proto.StellarSignMessage(address_n=address_n, message=message))
+
+    def stellar_verify_message(self, pubkey_bytes, signature, message):
+        resp = self.call(proto.StellarVerifyMessage(public_key=pubkey_bytes, message=message, signature=signature))
+
+        if isinstance(resp, proto.Success):
```

Comment:
nooooo, just `return isinstance(resp, proto.Success)` please .)
i know this is how all the other `verify_message` calls look, it's wrong in them too and I'll be changing it